Snakefile

```diff
@@ -681,15 +695,17 @@ rule plot_summary:
         "scripts/plot_summary.py"
 
 
-rule plot_network:
+rule build_test_configs:
```

Comment:
This rule may be generalized.
As input we may provide only two elements:
1. `base_config="config.tutorial.yaml"`: it represents base configuration file, hence the one called tutorial now
2. `file_changes_list=[...]`: a list of strings, each of them being the changes with respect to the tutorial. With respect to what already done I would mean: `["test/config.standard.yaml", ..., "test/tmp/config.landlock_tmp.yaml"]`

Then, in the build_test_configs, we can preload the base configuration as done now and loop over the `file_changes_list`.
For every file, we merge the changes into a brand new base configuration file and we can create the config file in the test folder as you are doing now.

I think this procedure is flexible and easy to extend with multiple test configurations beyond the existing one